tests/test_contrib/test_pytest_plugin/test_plugin.py

```diff
@@ -19,3 +22,44 @@ def test_call_pytest_setup_plan():
 
     assert 'migrator' in output_text
     assert 'migrator_factory' in output_text
+
+
+def test_pytest_registers_marker():
+    """Ensure ``MIGRATION_TEST_MARKER`` marker is registered."""
+    output_text = subprocess.check_output(
+        ['pytest', '--markers'],
+        stderr=subprocess.STDOUT,
+        universal_newlines=True,
+        encoding='utf8',
+    )
+
+    assert MIGRATION_TEST_MARKER in output_text
+
+
+def test_pytest_markers():
```

Comment:
@sobolevn what do you think about using [`pytester` plugin](https://docs.pytest.org/en/latest/writing_plugins.html#testing-plugins) to add more detailed tests of our pytest plugin?

Reply:
Yes, sure! Let's open a new issue for it! 👍

Reply:
I have added issue #140